### ISSUE TITLE
Create "Tutorial" in docs

### ISSUE THREAD

**Issue by Billy99** [bpfman/bpfman#153]
This issue covers the steps required to add a Tutorial to our documentation

**Comment by astoycos** [comment]
We should probably break this into writing a BPF program in Rust and C? The K8's demo I'll merge will show that, but it's using a version of the linker and aya that aren't available upstream :(

**Comment by Billy99** [comment]
Yes, and I wanted to go a little deeper, libbpf vs libxdp vs aya vs cilium library.

**Comment by astoycos** [comment]
Sweet! :) We also should start containerizing any and all examples as we go along and have a spot in Quay for them

**Comment by astoycos** [comment]
@Billy99 Has started this... Is it done in your opinion?

**Comment by Billy99** [comment]
This has started. Beefing up the text around the goCounter example and expanding the tutorial. But I think it needs more though.

**Comment by astoycos** [comment]
This may turn into a generic "lets improve docs" catcher for bpfd loading/unloading
